client/src/commands/run.ts

```diff
@@ -34,7 +36,14 @@ interface FoldingBlock {
   endCol: number;
 }
 
-let running = false;
+const { isRunning: initialRunState, setIsRunning } = runStore.getState();
+
+let isCodeRunning: boolean = initialRunState;
+
+subscribe(
+  (state) => state.isRunning,
+  (run) => (isCodeRunning = run),
```

Comment:
nit: just want to mention an alternative is to call `getState` each time, as it's not really reactive to do anything.

Reply:
Good feedback. Fixed in https://github.com/sassoftware/vscode-sas-extension/pull/713/commits/ad4374265039a8fb7dec2e66b729eee2cb994ede